subsys/infinispan/src/main/resources/infinispan.xml

```diff
@@ -13,6 +13,9 @@
 
     <local-cache name="koji-maven-version-metadata" >
       <eviction strategy="LRU" size="10000000" type="COUNT"/>
+      <persistence passivation="true">
```

Comment:
I think we should try persisting this into a postgres/other DB, but I'm not sure the filesystem persistence approach will work on a NFS volume. We'll have to be careful with this.

We ran into a lot of problems with filesystem persistence in the content indexing cache.

Reply:
I guess the critical path is reading from cache. We can config it "preload=true". Kuba said a fix for Indy 1.2.X should be sufficient. I suggest merging it to 1.2.x and master, and watch for performance/issue. If we want to be conservative (in case it may cause other bugs), we can hold the ISPN changes and merge the version-parsing fix only.